### PR TITLE
Copter: Determine the input value nan and inf

### PR DESCRIPTION
**Pull request by muramura**
I learned that the values of nan and inf are treated as normal values in MAVLINK communication from outside.
I thought it would be better to make it an abnormal value in MAVLINK communication.

### CODE REVIEW

File: ArduCopter/GCS_Mavlink.cpp [hunk 1]
```diff
@@ -582,6 +582,9 @@ MAV_RESULT GCS_MAVLINK_Copter::handle_command_int_packet(const mavlink_command_i
     switch(packet.command) {
     case MAV_CMD_DO_FOLLOW:
 #if MODE_FOLLOW_ENABLED == ENABLED
+        if (isnan(packet.param1) || isinf(packet.param1)) {
```
Comment (WickedShell) [inline ArduCopter/GCS_Mavlink.cpp]: Why bother checking this? You can't get into the accepted path here as one or both of the comparisons will fail, this just adds a different return value for a rare case, I don't think it's worth the runtime checks and code space.

File: ArduCopter/GCS_Mavlink.cpp [hunk 2]
```diff
@@ -672,6 +683,12 @@ MAV_RESULT GCS_MAVLINK_Copter::handle_command_long_packet(const mavlink_command_
         // param2 : speed during change [deg per second]
         // param3 : direction (-1:ccw, +1:cw)
         // param4 : relative offset (1) or absolute angle (0)
+        if (isnan(packet.param1) || isinf(packet.param1) ||
```
Comment (WickedShell) [inline ArduCopter/GCS_Mavlink.cpp]: Param1 checks here are redundant.

File: ArduCopter/GCS_Mavlink.cpp (hunk 1 above)
Comment (muramura) [reply ArduCopter/GCS_Mavlink.cpp]: @WickedShell san.
Thanks for your comment. I consider using AP_Mission :: sanity_check_params ().

**Comment by peterbarker** [comment]
@muramura did you wish to pursue this?  @WickedShell 's review comments haven't been addressed.

**Comment by peterbarker** [comment]
No answer in almost 2 years.

If you want to pursue this, please reopen and address Michael's comments.

**Pull request closed without merging**